src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,446 @@
+| plasma_radioactivity | Radioactivity in plasma | Unit of plasma radioactivity (for example, "kBq/mL") |
+
+### Metabolites
+
+This section may be omitted if metabolite measurements were not made.
+If metabolite measurements are available by manual sampling
+they should be added to the `*_recording-blood_discrete.tsv`
+and the values below should be added to the `*_recording-blood_discrete.json`.
+If metabolite measurements are available by an autosampler they should be added to the `*_recording-blood_continous.tsv`
+and the values below should be added to the `*_recording-blood_continous.json`.
+Hence, it may contain information regarding metabolite info, such as the following three column examples:
+
+| **Column name**            | **Description**                                                                   | **Units**                                                   |
+| -------------------------- | --------------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| metabolite_parent_fraction | Parent fraction of the radiotracer (0-1)                                          | Unit of parent fraction (for example, "unitless")           |
+| metabolite_polar_fraction  | Polar metabolite fraction of the radiotracer (0-1)                                | Unit of polar metabolite fraction (for example, "unitless") |
+| hplc_recovery_fractions    | HPLC recovery fractions (the fraction of activity that gets loaded onto the HPLC) | Unit of recovery fractions (for example, "unitless")        |
+
+### Radioactivity in whole blood
+
+This section may be omitted if whole blood measurements of radioactivity were not made.
+It contains information regarding sampled whole blood data.
+If whole blood measurements are available by manual sampling they should be added to the `*_recording-manual_blood.tsv`
+and the values below should be added to the `*_recording-manual_blood.json`.
+If whole blood measurements are available by an autosampler, they should be added to the `*_recording-autosampler_blood.tsv`
+and the values below should be added to the `*_recording-autosampler_blood.json`.
+
+| **Column name**           | **Description**                      | **Units**                                                                         |
+| ------------------------- | ------------------------------------ | --------------------------------------------------------------------------------- |
+| whole_blood_radioactivity | Radioactivity in whole blood samples | Unit of radioactivity measurements in whole blood samples (for example, "kBq/mL") |
+
+### Example (`*_recording-manual_blood.json`)
+
+```JSON
+{
+	"PlasmaAvail": true,
+	"WholeBloodAvail": true,
+	"MetaboliteAvail": true,
+	"MetaboliteMethod": "",
+	"MetaboliteRecoveryCorrectionApplied": false,
+	"DispersionCorrected": false,
+
+    "time": {
```

Comment:
The indentation and newlines in this snippet appear to be incompatible with JSONs.

Reply:
We are working on fixing that.